src/Acme.Umbraco.Models/Models/Athlete.generated.cs

```diff
@@ -22,7 +22,7 @@ namespace Umbraco.Web.PublishedModels
 {
 	/// <summary>Athlete</summary>
 	[PublishedModel("athlete")]
-	public partial class Athlete : PublishedContentModel
+	public partial class Athlete : PublishedContentModel, IContentBase, INavigationBase
```

Comment:
If this was a manual edit lets make a new class inherit from Athelete and Add the interfaces on that.